.git_config/hooks/pre-commit

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+STANDARD="PSR12"
```

Comment:
The coding standards to check should be `Drupal` & `DrupalPractice`. 
So, change this line to be like `STANDARD="Drupal,DrupalPractice"`

Reply:
OK. Will do the changes as requested. Thanks.